kakao-shopping/src/components/templates/OrderTemplate.jsx

```diff
@@ -0,0 +1,187 @@
+import { useMutation } from "react-query";
+import { getCart } from "../../apis/cart";
+import Loader from "../atoms/Loader";
+import { useState, useRef } from "react";
+import { useNavigate } from "react-router-dom";
+
+const OrderTemplate = ({ data }) => {
+  const { products, totalPrice } = data?.data?.response;
+  const naviagate = useNavigate();
+  const [allAgree, setAllagree] = useState(false);
+  const [agreePayment, setAgreePayment] = useState(false);
+  const [agreePolicy, setAgreePolicy] = useState(false);
+
+  const allAgreeRef = useRef(null);
+  const allPaymentRef = useRef(null);
+  const agreePolicyRef = useRef(null);
+
+  const handleAllAgree = (e) => {
+    const value = e.target.checked;
+    setAgreePayment(value);
+    setAgreePolicy(value);
+  };
+
+  const handleAgreement = (e) => {
+    const { name, checked } = e.target;
+
+    if (name === "payment-agree") {
+      setAgreePayment(checked);
+    } else if (name === "payment-agree") {
+      setAgreePolicy(checked);
+    }
+    if (!checked) {
+      setAllagree(false);
+    }
+  };
+
+  const {} = useMutation({
+    queryFn: () => order,
```

Comment:
```suggestion
    mutationFn: order,
``` 

처럼 적을 수 있을 것 같습니다!